src/main/resources/plugin.yml

```diff
@@ -18,5 +18,8 @@
     "damage": {
         description: "Stellt deinen Damager Schaden ein (in halben Herzen)"
     }
+      "randomfirework": {
```

Comment:
fix the wrong formatting by using ctrl + alt + L

Reply:
da fehlt auch ein komma